src/ui/atoms/listItem/ListItem.tsx

```diff
@@ -3,57 +3,79 @@ import type { DeepReadonly } from 'superTypes'
 import { Typography } from '../typography/Typography'
 import './listItem.scss'
 import { isString } from 'utils'
+import classNames from 'classnames'
+import type { ListProps } from 'ui/atoms/list/List'
 
-export type ListItemProps = {
+export type ListItemProps = Pick<ListProps, 'layout'> & {
   /**
    * The main title of the item.
    */
-  readonly title?: string
+  readonly title?: string | JSX.Element
   /**
    * Additional information about the item
    */
   readonly description?: string | JSX.Element
   /**
-   * An element that can provide visual information or allow interactions on the left side.
+   * An element that can provide visual information or allow interactions.
+   * The position of this element depends on the layout property value.
    */
-  readonly leftElement?: JSX.Element
+  readonly firstElement?: JSX.Element
   /**
-   * An element that can provide visual information or allow interactions on the right side.
+   * An element that can provide visual information or allow interactions.
+   * The position of this element depends on the layout property value.
```

Comment:
What's the purpose of repeating this same sentence two times (for `firstElement` and `lastElement`)?

Please, add some specific context about their spatialization because, when I read that, the two props do exactly the same thing.